flow/project.py

```diff
@@ -3576,6 +3583,13 @@ def submit(
             Additional keyword arguments forwarded to :meth:`~.ComputeEnvironment.submit`.
 
         """
+        if names is not None:
+            absent_ops = (set(self._groups.keys()) ^ set(names)) & set(names)
+            if absent_ops:
+                print(
```

Comment:
Should this go to stderr?

Reply:
I only noticed this inconsistency because the error message proposed by @csadorf is shorter, and thus triggers a reformat on this line which caused pre-checks to fail. Without specifying `file=sys.stderr`, the auto-formatting looks different for this line than the other messages. 😅

Reply:
The reason for choosing `sys.stderr` was partly because some of the warning statements (when printing them) were redirected to the error stream.

Reply:
@kidrahahjo I'm not sure if we were talking about the same thing. I fixed the thing I meant in ecf0d2b.